donkeycar/pipeline/types.py

```diff
@@ -71,27 +78,68 @@ class TubDataset(object):
     """
 
     def __init__(self, config: Config, tub_paths: List[str],
-                 shuffle: bool = True) -> None:
+                 seq_size: int = None) -> None:
```

Comment:
`None` is never a good default for an `int`. Either use `Option[int]` or just use a number like `-1` to represent a sentinel value.

Reply:
Ok, we can use `0` here for a non-defined value, as only positive integers make sense. Changed to that.